pom.xml

```diff
@@ -131,10 +148,72 @@
                         </instructions>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.owasp</groupId>
+                    <artifactId>dependency-check-maven</artifactId>
+                    <version>${dependency-check-maven.version}</version>
+                    <configuration>
+                        <!-- The following properties enable using a mirror for nist NVD data -->
+                        <cveUrl12Modified>${owasp.cveUrl12Modified}</cveUrl12Modified>
+                        <cveUrl20Modified>${owasp.cveUrl20Modified}</cveUrl20Modified>
+                        <cveUrl12Base>${owasp.cveUrl12Base}</cveUrl12Base>
+                        <cveUrl20Base>${owasp.cveUrl20Base}</cveUrl20Base>
+                        <!-- End NVD mirror configuration -->
+                        <failBuildOnCVSS>7</failBuildOnCVSS>
```

Comment:
✏️ This threshold should be much lower. We should fail on vulnerabilities scored as 2 or 3.

Reply:
Yeah as you can see I foolishly copied it from another project :blush: 
I'll set it to 2 and see if an execution turns up anything as well